From b0edcafe5f6c5b54f9360ef50f38458bb9407dd9 Mon Sep 17 00:00:00 2001
From: Matt Fleming <matt.fleming@intel.com>
Date: Mon, 2 Jun 2014 11:22:01 +0100
Subject: [PATCH] tool/setup.py: Delete Windows drivers in data_files

We obviously don't build those drivers on Linux, so remove them. To fix
this properly would require some means of telling setup.py which
platform we're building for, not which platform we're currently running
on.

Signed-off-by: Matt Fleming <matt.fleming@intel.com>
---
 source/tool/setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/source/tool/setup.py b/source/tool/setup.py
index 674e1f3..e88a454 100644
--- a/source/tool/setup.py
+++ b/source/tool/setup.py
@@ -74,7 +74,6 @@ setup(
         author_email    = '',
         url             = 'https://github.com/chipsec/chipsec',
 
-        data_files      = data_files,
         packages        = mypackages
 
 )
-- 
2.1.0

